clients/cli/tests/test.rs

```diff
@@ -142,7 +129,7 @@ async fn wait_for_next_epoch(rpc_client: &RpcClient) -> Epoch {
     println!("current epoch {}, advancing to next...", current_epoch);
     loop {
         let epoch_info = rpc_client.get_epoch_info().await.unwrap();
-        if epoch_info.epoch > current_epoch {
+        if epoch_info.epoch > current_epoch && epoch_info.slot_index > 0 {
             return epoch_info.epoch;
         }
```

Comment:
this was a frustrating but ultimately funny debugging experience, i was going nuts trying to figure out how my `command_deposit()` refactor introduced stake program error 10 (`VoteAddressMismatch`) in some tests and it took longer than id like to admit to notice it was actually error 0x10 (`EpochRewardsActive`)

i think something i changed made it slightly _faster_: before, by time the test transaction executed, a slot had already elapsed